files/pagerduty.rb

```diff
@@ -15,6 +15,7 @@ def api_key
   def trigger_incident
     return false unless api_key
     require 'redphone/pagerduty'
+
     response = Redphone::Pagerduty.trigger_incident(
       :service_key  => api_key,
       :incident_key => incident_key,
```

Comment:
doesn't `description` need to change here?

Reply:
I am sending in `:details => full_description_hash` below. I am not sure I understand what you mean by description needs to change here ?

Reply:
Part of the goal of the design was so that `event_description` would match up "perfectly" to the pagerduty description. Is that achieved with the change so far?

Reply:
@solarkennedy no, that has not been achieved. but the current call to description(140) appears as the title of the PagerDuty alert. And the description field that I have is just to add more context information to the event and does not serve as the title of the alert (in PagerDuty)

event_description is being passed down into details where it just serves as more context information for the event itself as intended.